providers/os/config/config.go

```diff
@@ -300,6 +300,12 @@ var Config = plugin.Provider{
 					Desc:   "Mount all partitions of the block device",
 					Option: plugin.FlagOption_Hidden,
 				},
+				{
+					Long:   "skip-attempt-expand-partitions",
```

Comment:
what would be the use case for that?

Reply:
skip mounting-remounting cycle, for example if we want just specific single partition maybe?